### ISSUE TITLE
CY channels

### ISSUE THREAD

**Issue by ROBGagn** [iptvnikolas/Nikki-TVM3U8#1]
Hello,

Some CY channels don't work (with or without VPN) in your file.

Alpha
Extra
Capital
Plus
Smile

Also do you have a link for RIK HD in 1080p as it is on cybc.com.cy website ? The one I have http://l6.cloudskep.com/tvb6/rikhd1/mpeg.2ts is only in 720p

Thanks :-)

**Comment by iptvnikolas** [comment]
All Cypriots work.

**Comment by iptvnikolas** [comment]
At the moment, however, Alpha is playing normally. I don't know the rest of the stations (Extra, Capital, Plus and Smile).

**Comment by ROBGagn** [comment]
Alpha is not working at all for me? I tried 3 different VPNs and not working...
Extra, Capital, Plus and Smile are not working. 
Do you have a better quality for RIK HD ? (1080p instead of 720p) ?

**Comment by iptvnikolas** [comment]
Yes. Alpha is working. Extra,Capital,Plus and Smile is lost and found of working.

**Comment by ROBGagn** [comment]
Your last updated link for Alpha TV CY is not working... 

http://l4.cloudskep.com/alphacyp/acy/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9Mi8xOS8yMDIzIDg6MzU6NTIgQU0maGFzaF92YWx1ZT1zMFB1ZVhZOGlSdU03YzRHOS9FVmN3PT0mdmFsaWRtaW51dGVzPTM2MA==

After some time the token expires...

**Comment by iptvnikolas** [comment]
They are some major temporary technical problems that Cypriot TV stations are facing in their live streams. I believe they will be back to normal soon.

**Comment by iptvnikolas** [comment]
But, I'm thinking of putting up an unofficial live stream of Alpha Cyprus.

**Comment by ROBGagn** [comment]
cool thanks.

**Comment by iptvnikolas** [comment]
The unofficial live stream of Alpha Cyprus is here:
https://l4.cloudskep.com/alphacyp/acy/playlist.m3u8?wmsAuthSign=c2VydmVyX3RpbWU9Mi8xOS8yMDIzIDg6MzU6NTIgQU0maGFzaF92YWx1ZT1zMFB1ZVMFH0FG1ZVMF000WRtaW51dGVzPTM2MA==

**Comment by ROBGagn** [comment]
Unfortunately, it's not working... the AuthSign token expires after a few minutes...

**Comment by iptvnikolas** [comment]
Maybe send an email to Alpha Cyprus and have them reopen its live stream? For years I had downloaded a corresponding application with TV channels on my tablet and all the Cypriot is working.

**Comment by iptvnikolas** [comment]
It is better to enter the Alpha Cyprus website and press F12 to get the link to m3u8 of the laptop.

**Comment by iptvnikolas** [comment]
I have the same issue with Omega and Sigma. The unofficial Alpha works fine but I can't copy it.

**Comment by ROBGagn** [comment]
Do you want to exchange links through PM or email ?

**Comment by iptvnikolas** [comment]
No
